README.md

```diff
@@ -534,20 +534,20 @@ The `content` section is the body of the post added in Markdown format.
 The `meta` section is a `YAML` like structure and should be wrapped with `---`
 signs. The meta section contains post-related information like:
 
-- `title` - Title of the blog post. Used also as the title of the page inside
+- `title (required)` - Title of the blog post. Used also as the title of the page inside
   `<title></title>` tag
 - `description` - Description of the blog post. Used inside `<meta name="description" />` tag
-- `published` - Publishing date of the blog post. Used also for sorting posts by
+- `published (required)` - Publishing date of the blog post. Used also for sorting posts by
   date (the format should be `Month d, yyyy` for example `January 1, 2023`)
-- `author` - Unique slug of the author (from Storyblok) usually looks like `jon-doe`.
+- `author (required)` - Unique slug of the author (from Storyblok) usually looks like `jon-doe`.
   Based on this property blog post page will display proper info about the author
   (and their avatar). The author must be present in Storyblok in order for the
   post to build without error.
 - `category` - Array of categories for example `[Machine Learning]`. All
```

Comment:
@gabalafou -- Do we know if this field is optional, or if it needs to be _present_, but can be an empty list?

Do we _want_ to allow someone not to declare a category?

Reply:
Honestly no idea. I don't think I've seen any blog post without a category though.

Reply:
For simplicity, I would just tell the blog author that all fields are required, even if that's not enforced by the machinery.